packages/statemanager/benchmarks/index.ts

```diff
@@ -0,0 +1,54 @@
+import { Account, Address, MapDB } from '@ethereumjs/util'
+// @ts-ignore - package has no types...
+import { run, mark, logMem } from 'micro-bmark'
+import { DefaultStateManager, FlatStateManager } from '@ethereumjs/statemanager'
```

Comment:
Just a suggestion but we already have `benchmark` used in a couple of places in the monorepo so not sure if it makes sense to use a different benchmarking tool here.  No strong opinion but just a thought (especially since I think the APIs are close to identical)

Reply:
I originally put this benchmark with the trie benchmarks, but later moved it over here since it was more related to the `statemanager` package. It uses the same benchmark tools as the trie benchmarks do (`micro-bmark`). The goal is to show the performance difference between the flat and default statemanagers.